code_annotations/find_annotations.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, config, source_path, report_path, verbosity):
             config: Configuration file path
             source_path: Directory to be searched for annotations
             report_path: Directory to write the report file to
-            verbosity: Integer representing verbosity level (1-3)
+            verbosity: Integer representing verbosity level (0-3)
```

Comment:
I was thinking of using VerboseEcho but ended up changing my mind.  I left in some changes.  In this case, it seems like the allowable levels should actually be 0-3, since setting it to 0 makes things even less verbose than 1, but not totally quiet.